templates/content-certificate.php

```diff
@@ -54,6 +54,10 @@
 				</button>
 			<?php endif; ?>
 
+			<?php if ( $is_sharing_enabled ) : ?>
+				<input disabled="disabled" name="llms_sharing_permalink" value="<?php echo esc_url( get_permalink( get_the_ID() ) ); ?>">
```

Comment:
Readonly in favor of disabled

A readonly field can still be tabbed into as opposed to a disabled field that cannot

```suggestion
				<input readonly="readonly" name="llms_sharing_permalink" value="<?php echo esc_url( get_permalink( get_the_ID() ) ); ?>">
```